### DISCUSSION TITLE
Windows 32 位程序访问Kimi 、文心一言等网站会不断刷新，但 64 位程序不会，什么原因？

### DISCUSSION THREAD

**Discussion by mogoweb** [CefView/QCefView#418] [Q&A]
碰到一个问题，使用 QCefViewTest （Windows 32-bit build） 访问 Kimi (https://kimi.moonshot.cn/)，文心一言(https://yiyan.baidu.com/) 等网站时，会不断刷新，但是使用 Windows 64-bit build，linux build 都没有问题。

开始怀疑是 CEF 32 位版本的问题，但是我使用CefSimple（32位）测试，也没有问题，这让我百思不得其解，看代码也没有什么不同啊。

这可能是哪里的问题。

排除网站的问题，使用Chrome、Edge、firefox 都没有此问题。

**Comment by tishion** [comment]
不断刷新是renderer进程在不断崩溃

**Reply by mogoweb** [reply]
对，我也发现是这个问题。我现在搞了一个临时解决方案，QCefView用32位的，但 CefViewWing 用 64 位的，但我不是很明白为啥32位会崩溃。如果是 Cef 本身的问题，但我用 CefSimple 测试，又没有问题
